src/main/java/com/dongbaeb/demo/profile/service/MemberService.java

```diff
@@ -0,0 +1,80 @@
+package com.dongbaeb.demo.profile.service;
+
+import com.dongbaeb.demo.entity.Member;
+import com.dongbaeb.demo.entity.University;
+import com.dongbaeb.demo.exception.ResourceNotFoundException;
+import com.dongbaeb.demo.profile.dto.MemberRequest;
+import com.dongbaeb.demo.profile.dto.MemberResponse;
+import com.dongbaeb.demo.profile.repository.MemberRepository;
+import com.dongbaeb.demo.profile.repository.UniversityRepository;
+import org.springframework.stereotype.Service;
+
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class MemberService {
+    private final MemberRepository memberRepository;
+    private final UniversityRepository universityRepository;
+
+    public MemberService(MemberRepository memberRepository, UniversityRepository universityRepository) {
+        this.memberRepository = memberRepository;
+        this.universityRepository = universityRepository;
+    }
+
+    @Transactional
+    public MemberResponse createMember(MemberRequest memberRequest) {
+        Member member = new Member();
+        member.setKakaoId(memberRequest.kakaoId());
+        member.setRole(memberRequest.role());
+        member.setName(memberRequest.name());
+        member.setNickname(memberRequest.nickname());
+        member.setProfileImageUrl(memberRequest.profileImageUrl());
+        member.setStudentNo(memberRequest.studentNo());
+
+        List<University> universities = memberRequest.universityIds().stream()
+                .map(universityRepository::findById)
+                .map(optionalUniversity -> optionalUniversity.orElseThrow(() -> new RuntimeException("University not found")))
+                .collect(Collectors.toList());
```

Comment:
스트림 활용 좋네요👍👍
하지만 예외는 멤버 예외와 같이 `ResourceNotFoundException` 으로 처리해 주고, 메시지도 한글로 작성해 주시면 좋을 것 같아요!

그리고 자바 17 부터는 다음과 같이 `toList()` 로 간편하게 처리 가능하니 참고하세욥!!
```suggestion
        List<University> universities = memberRequest.universityIds().stream()
                .map(universityRepository::findById)
                .map(optionalUniversity -> optionalUniversity.orElseThrow(() -> new RuntimeException("University not found")))
                .toList();
```